cxx-qt-gen/src/extract.rs

```diff
@@ -746,13 +678,14 @@ pub fn extract_qobject(module: &ItemMod) -> Result<QObject, TokenStream> {
     // The name of the Qt object we are creating
     let qt_ident = quote::format_ident!("{}", original_mod.ident.to_string().to_case(Case::Pascal));
 
-    // Extract the normal and invokables from the Parser methods
-    let extracted = extract_invokables(&qobject.methods, &qt_ident)?;
-
     // A list of the invokables for the struct
-    let object_invokables = extracted.invokables;
+    let object_invokables = qobject
+        .invokables
+        .iter()
+        .map(|invokable| extract_invokable(invokable, &qt_ident))
+        .collect::<Result<Vec<Invokable>, TokenStream>>()?;
```

Comment:
https://turbo.fish/::%3CResult%3CVec%3CInvokable%3E,%20TokenStream%3E%3E

Reply:
I suspect this line would look pretty ridiculous to someone who doesn't know Rust well :sweat_smile:

Reply:
Hehe, seems no worse than templating in C++ or Java :shrug: :-)